app/models/KijijiCar.py

```diff
@@ -0,0 +1,10 @@
+# Pydantic allows auto creation of JSON Schemas from models
+from pydantic import BaseModel
+
+
+class KijijiCar(BaseModel):
+  car_info: str
+  url: str
+  mileage: str
+  region: str
+  price: str
```

Comment:
For `mileage` and `price` in `usedCar`, it is int. Any reasons why this model should be str?

Reply:
Don't we need to show the unit?